ddtrace/tracer/tracer.go

```diff
@@ -306,6 +310,16 @@ func (t *tracer) worker(tick <-chan time.Time) {
 	}
 }
 
+func (t *tracer) writeTrace(trace []*span) {
+	if !t.runProcessor(trace) {
```

Comment:
Maybe you should call this `droppedByProcessor` and negate the return value. It will be self-documented then and would make more sense reading it as part of an `if` condition. You could then also remove the comment underneath. WDYT?

Reply:
Agreed this is better/ clearer. Changed this is latest commit 🙂